go.mod

```diff
@@ -16,19 +16,22 @@ require (
 	github.com/spf13/pflag v1.0.6
 	go.starlark.net v0.0.0-20231101134539-556fd59b42f6
 	golang.org/x/arch v0.11.0
-	golang.org/x/sys v0.26.0
-	golang.org/x/telemetry v0.0.0-20241106142447-58a1122356f5
-	golang.org/x/tools v0.26.0
+	golang.org/x/sys v0.35.0
+	golang.org/x/telemetry v0.0.0-20250807160809-1a19826ec488
+	golang.org/x/tools v0.36.0
 	gopkg.in/yaml.v3 v3.0.1
 )
 
 require (
 	github.com/cpuguy83/go-md2man/v2 v2.0.6 // indirect
+	github.com/fatih/color v1.18.0 // indirect
+	github.com/google/capslock v0.2.8 // indirect
```

Comment:
Unless I'm missing something capslock is never imported and the install script uses `@latest` so this shouldn't be needed, just like we don't depend on staticcheck.

Reply:
No you're right, I think this is leftover from when I was trying to "tool" approach, but that isn't supported until 1.24.